testgrid/metadata/job.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metadata
+
+// Started holds the started.json values of the build.
+type Started struct {
+	Timestamp   int64             `json:"timestamp"` // epoch seconds
+	RepoVersion string            `json:"repo-version"`
+	Node        string            `json:"node"`
+	Pull        string            `json:"pull"`
+	Repos       map[string]string `json:"repos"` // {repo: branch_or_pull} map
+}
+
+// Finished holds the finished.json values of the build
+type Finished struct {
+	// Timestamp is epoch seconds
+	Timestamp  int64    `json:"timestamp"`
+	Passed     bool     `json:"passed"`
+	JobVersion string   `json:"job-version"`
+	Revision   string   `json:"revision,omitempty"` // TODO(fejta): remove
```

Comment:
This should not be first-class

Reply:
this is actually repo-version equivalent - fyi I already messed with testgrid/gubernator to support revision, but feel free to change that again :man_shrugging: